src/Microsoft.ML.AutoML/ColumnInference/TextFileContents.cs

```diff
@@ -50,12 +50,14 @@ public static ColumnSplitResult TrySplitColumns(MLContext context, IMultiStreamS
         {
             var sparse = new[] { false, true };
             var quote = new[] { true, false };
+            var tryMultiline = new[] { false, true };
             var foundAny = false;
             var result = default(ColumnSplitResult);
             foreach (var perm in (from _allowSparse in sparse
                                     from _allowQuote in quote
                                     from _sep in separatorCandidates
-                                    select new { _allowSparse, _allowQuote, _sep }))
+                                    from _tryMultiline in tryMultiline
```

Comment:
naming pattern would be:
```suggestion
                                    from _allowMultiline in multiline
```